static/src/javascripts/projects/common/modules/commercial/contributions-utilities.js

```diff
@@ -78,6 +79,20 @@ const getVisitCount = (): number => local.get('gu.alreadyVisited') || 0;
 const replaceArticlesRead = (text: string, days: number = 30): string =>
     text.replace(/%%ARTICLES_READ%%/g, `${getArticleViewCount(days)}`);
 
+const replaceCountryName = (text: string): string => {
+    const countryName = countryNames[geolocationGetSync()];
+    if (countryName) {
+        return text.replace(/%%COUNTRY_NAME%%/g, countryName);
+    }
+    return text;
```

Comment:
We should return some default which replaces `%%COUNTRY_NAME%%` name with _something_ so we don't return this variable to readers outside of the list

Reply:
it shouldn't be possible to be put in the test if `countryNameIsValid` is false, but you're absolutely right that it should be more defensive than this - will default to GB

Reply:
what actually happens if the country isn't in the list? Is it the job of the test to limit it to the pre-defined list of countries? 

If defaulting to GB means showing the wrong country to anyone, I'm not sure we should do that really.

Reply:
the test will never be shown to users not in that list of countries. Defaulting to GB is being extra defensive in case something goes wrong

Reply:
I've just rewritten it to avoid having a default. Basically, as long as `hasCountryName` is set, users will only be opted into the test if they're in a valid country